File: NET/com.yupanastudio.igest.winapp.model/admin/Usuario.cs
using System.Collections.Generic;

namespace com.yupanastudio.igest.winapp
{
    public class Usuario
    {
        public long idUsuario { get; set; }
        public string nomUsu { get; set; }
        public string pwdUsu { get; set; }
        public bool activo { get; set; }
        public bool cambioPwd { get; set; }
        public Persona obPersona { get; set; }
        public List<UsuarioSedeSistema> lsUsuarioSedeSistema { get; set; }
        public List<Perfil> lsPerfil { get; set; }

    }//end class

}

File: FLUTTER/igest_app/lib/models/principal/usuario.dart
import 'package:igest_app/models/principal/persona.dart';

class Usuario {
  int? idUsuario;
  String? nomUsu;
  String? pwdUsu;
  bool? activo;
  bool? cambioPwd;

  Persona? obPersona;
  //List<UsuarioSedeSistema>? lsUsuarioSedeSistema;
  //List<Perfil>? lsPerfil;

  Usuario({this.idUsuario, this.nomUsu, this.pwdUsu, this.activo, this.cambioPwd, this.obPersona});

}
